nhsn/tests/test_patch.py

```diff
@@ -13,7 +13,7 @@
 from delphi_nhsn.patch import filter_source_files, patch
 from delphi_nhsn.constants import TOTAL_ADMISSION_COVID_COL, TOTAL_ADMISSION_FLU_COL, \
     NUM_HOSP_REPORTING_FLU_COL, NUM_HOSP_REPORTING_COVID_COL, GEOS, TOTAL_ADMISSION_COVID, TOTAL_ADMISSION_FLU, \
-    NUM_HOSP_REPORTING_COVID, NUM_HOSP_REPORTING_FLU
+    NUM_HOSP_REPORTING_COVID, NUM_HOSP_REPORTING_FLU, NUM_HOSP_REPORTING_RSV_COL, TOTAL_ADMISSION_RSV_COL
```

Comment:
suggestion: we don't seem to use any of the non-"_COL" `NUM_HOSP_REPORTING...` so remove those. I was confused why we weren't adding a "num_hosp_reporting" for RSV too.

```suggestion
    NUM_HOSP_REPORTING_RSV_COL, TOTAL_ADMISSION_RSV_COL
```

Reply:
It was leftover from when i changed the variable names...